ou_dedetai/main.py

```diff
@@ -202,9 +178,23 @@ def get_parser():
         # help='check resources'
         help=argparse.SUPPRESS,
     )
+    cmd.add_argument(
```

Comment:
Do we need these arguments (the ones to basically run wine or winetricks)? There are already syslinks in standard locations if we need to debug using them.

Reply:
Having them as command args makes it much easier to run them within the correct wine environment. If someone wants to run them directly from the terminal they have to first set `WINEPREFIX`, possibly `WINELOADER`, `WINE`, and `WINESERVER`, and potentially `WINEDLLOVERRIDES` in the env, depending on what they are trying to accomplish.

Reply:
I'd almost rather support a command that exports those than wrapping wine/winetricks. For example (something like): 
```bash
source `our_binary --envs`
```

This is fine for now, we may want to consider if we want to maintain this for v1